docs/source/user_guide/integrations/aws.rst

```diff
@@ -0,0 +1,53 @@
+Amazon Web Services
+===============
```

Comment:
In RST, you have to make the `====` span the entire title haha.

Locally you can build your docs with:

```
cd docs/
rm -rf build/
make html
```

And you should be able to see/fix any warnings that pop up